src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -1877,27 +1980,47 @@ components:
         subject:
           type: string
           description: The subject of the qualification.
+          example: Maths GCSE
         level:
-          type: string
-          description: The level of the qualification (if known/relevant).
-          enum:
-            - ENTRY_LEVEL
-            - LEVEL_1
-            - LEVEL_2
-            - LEVEL_3
-            - LEVEL_4
-            - LEVEL_5
-            - LEVEL_6
-            - LEVEL_7
-            - LEVEL_8
+          $ref: '#/components/schemas/QualificationLevel'
         grade:
           type: string
-          description: The grade which was achieved (if known/relevant).
+          description: |
+            The grade which was achieved (if known/relevant).  
+            Note: This is a free format value and there is no type or enum. Therefore values can be "A", "B", "C" etc,
+            but also "1", "2", "3", "Pass", "Distinction", "Merit", "First class honours" etc. It is up to the consumer
+            to interpret this data as necessary.
+          example: Distinction
       required:
         - subject
         - level
         - grade
 
+    QualificationLevel:
```

Comment:
I've moved this out of `AchievedQualification` and made it it's own enum type; to prevent it being duplicated in both `AchievedQualification`  and `AchievedQualificationResponse`